File: duna-core/src/main/java/com/google/inject/UnsafeTypeLiteral.java
/*
 * Copyright (c) 2017 Duna Open Source Project
 * Ministério do Planejamento, Desenvolvimento de Gestão
 * República Federativa do Brasil
 *
 * This file is part of the Duna Project.
 */
package com.google.inject;

import java.lang.reflect.Type;

/**
 * A type literal from an unchecked type.
 */
public class UnsafeTypeLiteral extends TypeLiteral<Object> {

    public UnsafeTypeLiteral(Type type) {
        super(type);
    }
}

File: duna-core/src/main/kotlin/com/google/inject/UnsafeTypeLiteral.kt
/*
 * Copyright (c) 2016 Duna Project
 * Ministério do Planejamento, Desenvolvimento de Gestão
 * República Federativa do Brasil
 *
 * This file is part of the Duna Project.
 */
package com.google.inject

import java.lang.reflect.Type

/**
 * A type literal from an unchecked type.
 */
internal class UnsafeTypeLiteral(type: Type) : TypeLiteral<Any>(type)
